hist/histv7/inc/ROOT/RHist.hxx

```diff
@@ -333,6 +340,81 @@ void Add(RHist<DIMENSIONS, PRECISION, STAT_TO...> &to, const RHist<DIMENSIONS, P
    toImpl.GetStat().Add(fromImpl.GetStat());
 }
 
+/// Divide the `to` histogram by `from`.
+///
+/// `to = to/from`
+/// This operation may currently only be performed if the two histograms have
+/// the same axis configuration, use the same precision, and if `from` records
+/// at least the same statistics as `to` (recording more stats is fine).
+///
+/// Dividing histograms with incompatible axis binning will be reported at runtime
+/// with an `std::runtime_error`. Insufficient statistics in the source
```

Comment:
"Insufficient statistics" will be misinterpreted by physicists as "low bin content". And I bet that's not what you mean :-) What do you mean, can you reword?